src/OSPSuite.Core/Domain/Services/MoleculePropertiesContainerTask.cs

```diff
@@ -58,11 +61,11 @@ public IContainer CreateGlobalMoleculeContainerFor(IContainer rootContainer, Mol
          var globalMoleculeContainer = addContainerUnder(rootContainer, moleculeBuilder, moleculeBuilder.Name, simulationBuilder)
             .WithContainerType(ContainerType.Molecule);
 
+         var lastSpatialStructureGlobalMoleculeContainer = simulationBuilder.SpatialStructures.LastOrDefault()?.GlobalMoleculeDependentProperties;
```

Comment:
I am not sure if we always will have one container or not. The code before assumed yes but this may have been wrong